app/controllers/evidence_controller.rb

```diff
@@ -108,12 +110,18 @@ def autogenerate_issue
   end
 
   def liquid_resource_assigns
+    # find evidence inline for preview action
+    @evidence = @evidence || @node.evidence.find_by(id: params[:id])
     {
       'evidence' => EvidenceDrop.new(@evidence),
-      'node' => NodeDrop.new(@evidence.node)
+      'node' => NodeDrop.new(@node)
     }
   end
 
+  def set_form_preview_path
+    @form_preview_path = preview_project_node_evidence_path(current_project, @node, @evidence)
```

Comment:
I'm sure you can metaprogram this one, but hey 11th hour and all that.

Reply:
Updated to use a single method in `Previewable`